vision/object_detection_segmentation/yolov4/README.md

```diff
@@ -100,7 +100,7 @@ def postprocess_bbbox(pred_bbox, ANCHORS, STRIDES, XYSCALE=[1,1,1]):
         xy_grid = np.tile(np.expand_dims(xy_grid, axis=0), [1, 1, 1, 3, 1])
         xy_grid = xy_grid.astype(np.float)
 
-        pred_xy = ((scipy.special.expit(conv_raw_dxdy) * XYSCALE[i]) - 0.5 * (XYSCALE[i] - 1) + xy_grid) * STRIDES[i]
+        pred_xy = ((special.expit(conv_raw_dxdy) * XYSCALE[i]) - 0.5 * (XYSCALE[i] - 1) + xy_grid) * STRIDES[i]
```

Comment:
Is this import change necessary? It seems less readable than before.

Reply:
https://stackoverflow.com/questions/9819733/scipy-special-import-issue 
After looking into it, I see why this was necessary! Thank you for the fix.